utah-design-system-website/src/react/components/websiteContent/guidelines/AccessibilityChecklistDocumentation.jsx

```diff
@@ -47,21 +53,32 @@ function AccessibilityTestingDocumentation() {
                 </li>
               </ul>
             </li>
-            <li>The majority of body text is a minimum of 16px (1rem)</li>
+            <li>The majority of <Link to={pageUrls.typography}>body text</Link> is a minimum of 16px (1rem)</li>
+            <li><Link to={pageUrls.typography}>Text</Link> is in plain language that is understandable at an 8th grade level.</li>
             <li>
-              Headings (h1, h2, h3, h4, h5, h6) are sequential and distinct when compared to the body text.
+              <Link to={pageUrls.headings}>Headings</Link> (<code>h1, h2, h3, h4, h5, h6</code>) create an outline for the page content. They are sequential and clearly distinct from the body text.
               (Don’t skip heading levels or go out of order.)
             </li>
-            <li>Text is in plain language that is understandable at an 8th grade level.</li>
-            <li>Hyperlinks do not include generic text such as &quot;click here&quot; or &quot;view more&quot;.</li>
-            <li>Hyperlinks that open a new tab have a visual indicator and visually hidden text like “opens in a new tab” to warn screen reader users.</li>
+            <li><Link to={pageUrls.lists}>Lists</Link> (<code>ul, ol</code>) are used to identify all content that can be described as a list of something.</li>
+            <li><Link to={pageUrls.links}>Links</Link> do not include generic text such as &quot;click here&quot; or &quot;view more&quot;.</li>
+            <li>Links that open a new tab have a visual indicator and visually hidden text like “opens in a new tab” to warn screen reader users.</li>
+            <li>Links to phone numbers and email address should be readable by screen readers.</li>
             <li>Provide a <Link to={pageUrls.skipLink}>Skip Link</Link> at the top of all pages to aid those using assistive technology navigate your site.</li>
+            <li>
+              <Link to={pageUrls.table}>Tables</Link> should used for presenting rows and columns of data.
+              <ul>
+                <li>Never use tables for layout.</li>
+                <li>
+                  Column and row headers must be correctly identified.
+                </li>
+              </ul>
+            </li>
           </ul>
         </li>
         <li>
           Color
           <ul>
-            <li>Text, including hyperlinks, has a <code>4.5:1</code> contrast ratio to its background.</li>
+            <li>Text, including links, has a <code>4.5:1</code> <Link to={`${pageUrls.accessibility}#section-limited-vision-contrast`}>contrast ratio</Link> to its background.</li>
```

Comment:
just a thought, if the accessibility page ever had the id changed for `section-limited-vision`, this link would not go to that section. Would we want to use constants for these linked ids?

Reply:
Thats a lot of constants. Thats a lot of finding these.

UDS-1293